packages/compose/README.md

```diff
@@ -315,7 +315,7 @@ const WithFocusOnMount = () => {
 
 _Parameters_
 
--   _focusOnMount_ `boolean | 'firstElement'`: Focus on mount mode.
+-   _focusOnMount_ `boolean | 'firstElement' | Function`: Focus on mount mode.
```

Comment:
This doesn't work and I get a lint warning. How do I fix that? This isn't TypeScript...